README.md

```diff
@@ -0,0 +1,91 @@
+# is-dom-node
+
+![Build Status](https://img.shields.io/travis/cjbarth/is-dom-node)
+![Code Coverage](https://img.shields.io/codecov/c/github/cjbarth/is-dom-node)
```

Comment:
Should we configure those to point to org based accounts?

Reply:
Yes.